.editorconfig

```diff
@@ -1,4 +1,5 @@
 root = true
 
 [*.{cs,xaml}]
-indent_style = tab
\ No newline at end of file
+indent_style = tab
```

Comment:
I'm guessing changes to .editorconfig weren't meant to be in the PR.

Reply:
@hartez  I'll remove this specific confusing change but I did intend to modify editor config to include
```
csharp_space_after_keywords_in_control_flow_statements = true
```
After @samhouts  had pointed out some incorrect spacing that I was using.    Is that ok to leave that in for this PR?

Reply:
I'd rather have that in a separate commit.

Reply:
sounds good created a different PR for that